workflows/config/snakemake/config.yaml

```diff
@@ -1,15 +1,15 @@
-"seed": 123
+"seed": 12345
 "population_id": 0
 "num_samples_per_population": [10, 10, 10]
 "replicates": 1
 
 "species": "HomSap"
 "demo_models": [
     {"id":"Constant", 
-    "num_samples_per_population": [10],
+    "num_samples_per_population": [100],
     },
     {"id":"OutOfAfrica_3G09",
-    "num_samples_per_population": [10,10, 10],
+    "num_samples_per_population": [100,100,100],
     }
```

Comment:
I think these sample sizes should be left small be default, just while we are debugging the pipeline. alternatively you might add a second config file called like `production-config.yml` that has these large sample sizes

Reply:
I changed the name of `inference.snake` to `dfe.snake` and added `production-config.yml` with larger sample size.